website/addons/base/views.py

```diff
@@ -516,21 +550,21 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
         raise HTTPError(httplib.BAD_REQUEST)
 
     if not isinstance(node_addon, StorageAddonBase):
-        raise HTTPError(httplib.BAD_REQUEST, {
+        raise HTTPError(httplib.BAD_REQUEST, data={
             'message_short': 'Bad Request',
-            'message_long': 'The add-on containing this file is no longer connected to the {}.'.format(node.project_or_component)
+            'message_long': 'The add-on {} containing {} is no longer connected to {}.'.format(provider, path, node.project_or_component)
```

Comment:
Minor copyedit: for this and the others below, can you change the wording to "The {} add-on containing"?  I think "The github add-on containing foo.txt" sounds better than "The add-on github containing foo.txt".